vm/native/native.cpp

```diff
@@ -1,4 +1,6 @@
 //#define GC_DEBUG
+//#define GC_MAXIMUM_HEAP_SIZE (128 * 1024 * 1024)
```

Comment:
After #1763, there'll be a Make variable, *ASMJS_TOTAL_MEMORY*, that defines this constant. I'm unsure if it's worth preprocessing this file just for that variable, although something to consider.